static/src/javascripts/projects/common/modules/gallery/lightbox.js

```diff
@@ -295,16 +297,97 @@ class GalleryLightbox {
         this.fsm.trigger(event, data);
     }
 
-    loadGalleryfromJson(galleryJson: GalleryJson, startIndex: number): void {
-        this.index = startIndex;
+    static loadNextOrPrevious(
+        currentImageId: string,
+        direction: string
+    ): Promise<Object> {
+        const pathPrefix = direction === 'forwards' ? 'getnext' : 'getprev';
+        const seriesTag = config
+            .get('page.nonKeywordTagIds')
+            .split(',')
+            .filter(tag => tag.includes('series'))[0];
+        const fetchUrl = `/${pathPrefix}/${seriesTag}/${currentImageId}`;
+        return fetch(fetchUrl)
+            .then(response => response.json())
+            .then(json =>
+                // filter out non-lightbox items in the series
+                json.filter(image => image.images.length > 0)
+            );
+    }
 
-        if (this.galleryJson && galleryJson.id === this.galleryJson.id) {
+    loadOrOpen(newGalleryJson: Object): void {
```

Comment:
Is it worth typing this to avoid using `Object`?

We have the `GalleryJson` type that is used in this file, hopefully this overlaps with the structure of the newGalleryJson? 😁😁

```
type GalleryJson = {
    id: string,
    headline: string,
    shouldHideAdverts: boolean,
    standfirst: string,
    images: Array<ImageJson>,
};
```